src/pages/UploadPage/SingleFileUpload/FileHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from '@material-ui/core';
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import InsertDriveFileIcon from '@material-ui/icons/InsertDriveFile';
+import CardContent from '@mui/material/CardContent';
```

Comment:
Hey, we don't use @material-ui. We use @mui-ui. We were talking about it several times